target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8064.dtsi

```diff
@@ -633,7 +633,7 @@
 				mux {
 					pins = "gpio3";
 					function = "pcie1_rst";
-					drive-strength = <2>;
+					drive-strength = <12>;
```

Comment:
2 -> 12 doesn't look like `add missing gpio and gsbi declaration`, probably separate commit with description why we need to change 2 to 12.

Reply:
Added a reason in separate commit